rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -33,11 +33,23 @@ def getImageUploadAndColor():
         print(request.get_header(head))
         if headers.get(head) not in request.get_header(head):
             abort(406, 'Incorrect headers')
+    
+    colors = ['green', 'blue', 'yellow', 'red', 'purple', 'pink', 'black', 'brown', 'dark red', 'dark yellow', 'dark blue',
+    'light red', 'light green', 'grey', 'orange', 'white']
```

Comment:
Move this up above the method name. It probably won't be just for this method, so I would rather have it in the scope of the file instead of the scope of the method.